src/components/HeroBanner/index.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { Hyperlink } from '@edx/paragon';
+import './HeroBanner.scss';
+
+
+const ViewOrPurchaseButton = ({
+  authorized,
+  journalAboutId,
+  purchaseUrl,
+  price,
+}) => (
+  authorized ? (
+    <Link to={{ pathname: `/${journalAboutId}` }} className="btn btn-success">View Now</Link>
+  ) : (
+    <Hyperlink destination={purchaseUrl} className="btn btn-success" content={`Purchase Access ($${price})`} />
+  )
+);
+
+const HeroBanner = props => (
+  <div className="hero" style={{ background: `linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5) ), url(${props.bannerUrl})` }} >
+    <div><h1>{props.title}</h1></div>
```

Comment:
It feels weird to have `<h1>`s and `<h2>`s in `<div>`s. If it due to the `.hero > div { margin: 10px 20px; }` CSS, you can just use `.hero > *` instead.